include/repeated_hafnian.hpp

```diff
@@ -355,19 +450,20 @@ std::complex<double> loop_hafnian_rpt_quad(std::vector<std::complex<double>> &ma
  *      `mat = [1]` and `rpt = [6]` represents a \f$6\times 6\f$ matrix of all ones.
  * @return loop hafnian of the input matrix
  */
-double loop_hafnian_rpt_quad(std::vector<double> &mat, std::vector<double> &mu, std::vector<int> &rpt) {
-    std::vector<long double> matq(mat.begin(), mat.end());
-    std::vector<long double> muq(mu.begin(), mu.end());
-    long double haf;
-    int s = std::accumulate(rpt.begin(), rpt.end(), 0);
-    int n = std::sqrt(static_cast<double>(mat.size()));
-
-    if ( s == 0 || n == 0)
-        haf = 1.0;
-    else
-        haf = loop_hafnian_rpt(matq, muq, rpt);
-
-    return static_cast<double>(haf);
+double loop_hafnian_rpt_quad(std::vector<double> &mat, std::vector<double> &mu,
+                             std::vector<int> &rpt) {
+  std::vector<long double> matq(mat.begin(), mat.end());
+  std::vector<long double> muq(mu.begin(), mu.end());
+  long double haf;
+  int s = std::accumulate(rpt.begin(), rpt.end(), 0);
+  int n = std::sqrt(static_cast<double>(mat.size()));
+
+  if (s == 0 || n == 0)
+    haf = 1.0;
+  else
+    haf = loop_hafnian_rpt(matq, muq, rpt);
+
+  return static_cast<double>(haf);
 }
 
-}
+} // namespace libwalrus
```

Comment:
How come this comment is here? Just as a reminder? Almost thought it was an accidental namespace declaration :laughing:

Reply:
I think that comment is just to indicate that the functions of the namespace libwalrus end there. I quite like it since I always find it confusing because the previous function block is not indented.

Reply:
I agree!